pronotepy/clients.py

```diff
@@ -509,24 +510,65 @@ def homework(
                 out.append(hw)
         return out
 
-    def messages(self) -> List[dataClasses.Message]:
+    def get_recipients(self) -> List[dict]:
```

Comment:
I think we should return a `Recipient` object instead of a raw dict if it is meant for public use.

Reply:
I thought it was a bit like taking out the heavy weapon for nothing, but why not?

Reply:
I haven't seen the dict but nonetheless it really should be nicely wrapped somehow. Technically a dict can be whatever, having a class for it will make it more consistent.